### PR TITLE
(1) Theme: change color layout.

### PR DESCRIPTION
**Pull request by janiversen**
### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [ x] New feature
- [ ] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### Changes made:

Show the user all colours:

Captura de pantalla 2020-01-30 a las 18 09 06

### CODE REVIEW

File: mobile-widgets/qml/TemplateButton.qml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: GPL-2.0
```
Comment (dirkhh) [inline mobile-widgets/qml/TemplateButton.qml]: Can we do a single commit that does indeed do the git move and replaces all the uses?
I think that would be preferrable.

File: mobile-widgets/themeinterface.cpp [hunk 2]
```diff
@@ -26,6 +26,7 @@ void themeInterface::setup(QQmlContext *ct)
 	instance()->set_currentScale(qPrefDisplay::mobile_scale());
 }
 
+
```
Comment (dirkhh) [inline mobile-widgets/themeinterface.cpp]: that's an interesting commit. I think my small cleanup caused your tooling to add this...
obviously this commit should be dropped

File: mobile-widgets/themeinterface.cpp [hunk 3]
```diff
@@ -26,7 +26,6 @@ void themeInterface::setup(QQmlContext *ct)
 	instance()->set_currentScale(qPrefDisplay::mobile_scale());
 }
 
-
```
Comment (dirkhh) [inline mobile-widgets/themeinterface.cpp]: and that's the other half of it. Please drop as well.

File: mobile-widgets/qml/Settings.qml [hunk 4]
```diff
@@ -139,201 +139,221 @@ TemplatePage {
...
+					currentIndex: (ThemeNew.currentTheme === "Blue") ? 0 :
+								  (ThemeNew.currentTheme === "Pink") ? 1 : 2
+					model: ListModel {
+						   ListElement {text: qsTr("Blue")}
+						   ListElement {text: qsTr("Pink")}
+						   ListElement {text: qsTr("Dark")}
+					}
+					onActivated:  {
+						ThemeNew.currentTheme = currentIndex === 0 ? "Blue" :
+												currentIndex === 1 ? "Pink" : "Dark"
```
Comment (dirkhh) [inline mobile-widgets/qml/Settings.qml:157]: odd whitespace

**Review by dirkhh — COMMENTED** [review]
a couple of small suggestions for cleanup. but overall good

File: mobile-widgets/qml/Settings.qml (hunk 4 above)
Comment (janiversen) [reply mobile-widgets/qml/Settings.qml:157]: line 149 is alignment (at least in Qt Creator)
line 151-153 is also alignment

Which I thought to be ok (at least you have corrected me, for changing that earlier).

File: mobile-widgets/qml/TemplateButton.qml (hunk 1 above)
Comment (janiversen) [reply mobile-widgets/qml/TemplateButton.qml]: Done.

**Comment by janiversen** [comment]
Pushed new version, that hopefully contain all the changes you requested.

**Comment by janiversen** [comment]
Seems I managed to break something by removing the 2 commits, will look at it tomorrow

**Comment by dirkhh** [comment]
looks like you still have a dependency on the `SsrfButton.qml` in your `.qrc` file.

**Comment by janiversen** [comment]
Could not sleep (which is bad because I have a 95m dive tomorrow or Saturday depending on the current), kept thinking about this error. Thanks for pointing me in the right direction.

Force pushed an update, lets see if it compiles, and then back to the sack. It going to be an interesting dive  2x 3mix using sidemount, 1x Air transport (up/down) and 1x 85% O2 for deco. Going in a group of 5.

**Comment by janiversen** [comment]
Looks good, at least no general errors. Going zzz offline.

**Comment by janiversen** [comment]
I assume this PR does not really make sense anymore. I will let the branch stay for a couple of days should I be wrong in my assumption.

**Pull request closed without merging**